ci/provisioning/post_provision_config_nodes_LEAP_15.sh

```diff
@@ -104,11 +91,6 @@ post_provision_config_nodes() {
         exit 1
     fi
 
-    if ! time dnf -y install "${PYTHON_MACROS_RPM[@]}" ; then
-        dump_repos
-        exit 1
-    fi
-
     cat /etc/do-release
```

Comment:
Since only our CI nodes will have /etc/do-release, maybe in the future we should first test if this file exist.